### PR TITLE
add detection output api

### LINKED ISSUE #8380: Add detection output python api
Related #7402

### PR DESCRIPTION
**Pull request by Noplz**
Fix #8380

### CODE REVIEW

File: python/paddle/v2/fluid/tests/test_layers.py [hunk 1]
```diff
@@ -309,6 +309,34 @@ def test_multiplex(self):
             self.assertIsNotNone(out)
         print(str(program))
 
+    def test_detection_output(self):
```
Comment (qingqing01) [inline python/paddle/v2/fluid/tests/test_layers.py]: Write test in a new file: `tests/test_detection.py`.

File: python/paddle/v2/fluid/layers/detection.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+# limitations under the License.
+"""
+All layers just related to the detection neural network.
+"""
+
+from ..layer_helper import LayerHelper
+from ..initializer import Normal, Constant
+from ..framework import Variable
+from ..param_attr import ParamAttr
+from layer_function_generator import autodoc
```
Comment (qingqing01) [inline python/paddle/v2/fluid/layers/detection.py]: Remove the unused imports from lin 19 to line 22.

File: python/paddle/v2/fluid/tests/test_layers.py (hunk 1 above)
Comment (Noplz) [reply python/paddle/v2/fluid/tests/test_layers.py]: Done

File: python/paddle/v2/fluid/layers/detection.py (hunk 2 above)
Comment (Noplz) [reply python/paddle/v2/fluid/layers/detection.py]: Done

**Pull request closed without merging**